docs/native/src/wopi_requirements.rst

```diff
@@ -5,7 +5,7 @@ WOPI implementation requirements for |Office iOS| integration
 =============================================================
```

Comment:
I'll go ahead and list this as a requirement on the WOPI requirements again, and remove the references for alternate support in the operational requirements page.